ide/Renesas/e2studio/RX72N/wolfssl_Rx72N/user_settings.h

```diff
@@ -0,0 +1,84 @@
+/* user_settings.h
+ *
+ * Copyright (C) 2014-2025 wolfSSL Inc.
+ *
+ * This file is part of wolfSSH.
+ *
+ * wolfSSH is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSSH is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with wolfSSH.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#define WOLFCRYPT_ONLY
+#define NO_ERROR_STRINGS
+
+#define NO_MAIN_DRIVER
+#define NO_WRITEV
+#define NO_DEV_RANDOM
+#define NO_WOLFSSL_DIR
+#define NO_WOLFSSL_STUB
+/* for compilers not allowed dynamic size array */
+#define NO_DYNAMIC_ARRAY
+#define NO_RC4
+#define NO_OLD_SHA256
+#define NO_PWDBASED
+#define NO_PKCS12
+#define NO_PKCS8
+#define NO_DES3
+#define NO_MD4
+#define NO_FILESYSTEM
+#define WOLFSSL_NO_CURRDIR
+#define WOLFSSL_LOG_PRINTF
+#define WOLFSSL_SMALL_STACK
+#define WOLFSSL_DH_CONST
+#define WOLFSSL_USER_IO
+
+#define TFM_TIMING_RESISTANT
+#define ECC_TIMING_RESISTANT
+#define WC_RSA_BLINDING
+
+#define FP_MAX_BITS   4096
```

Comment:
With SP math FP_MAX_BITS is not used... You need `SP_INT_BITS 2048` which is likely default. FP_MAX_BITS only applies to USE_FAST_MATH and only if WOLFSSL_SP_MATH is not defined.